packages/ui/src/Radio.module.scss

```diff
@@ -0,0 +1,89 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.wrapper {
+  display: inline-flex;
+  // inline-flex changes vertical align compared to display: flex
+  // setting vertical-align allows us to easily use radio button
+  // inline e.g. next to the text
+  // Most common use case will be still the usage in a form where parent
+  // wrapper controls this.
+  vertical-align: bottom;
+  flex-direction: row;
+  user-select: none;
```

Comment:
This was also discussed with me and @aigoncharov before, it's a matter of preference and I've found quite a lot of tutorials where they do it, but since this is the second question about it I suggest we remove it and see how it goes.